index.js

```diff
@@ -309,15 +316,15 @@
 
       return bbox
     }
-    
+
     // Private - replace D3JS 3.X d3.functor() function
     function functor(v) {
-    	return typeof v === "function" ? v : function() {
+      return typeof v === 'function' ? v : function() {
         return v
-    	}
+      }
     }
 
     return tip
-  };
-
+  }
+// eslint-disable-next-line semi
```

Comment:
the semi-colon should be at the beginning of the iife

```js
;(function () {
  // code
})()
```

See the third exemple: http://eslint.org/docs/rules/semi#never